Common/GPU/Vulkan/VulkanDebug.cpp

```diff
@@ -49,6 +61,14 @@ VKAPI_ATTR VkBool32 VKAPI_CALL VulkanDebugUtilsCallback(
 		return false;
 	}
 
+	{
+		std::lock_guard<std::mutex> lock(g_errorCountMutex);
+		if (g_errorCount[messageCode] > 10) {
+			WARN_LOG(G3D, "Too many validation messages with message %d", messageCode);
```

Comment:
Should there be a `return false;` here?

-[Unknown]

Reply:
Right, fixed up the logic properly.